ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -299,6 +404,30 @@ def test_analytic_udf_destruct(backend, alltypes):
     backend.assert_frame_equal(result, expected)
 
 
+@pytest.mark.only_on_backends(['pandas'])
```

Comment:
I think should work work on pyspark as well

Reply:
This test in pyspark actually fails with 
`ibis.common.exceptions.OperationNotDefinedError: No translation rule for <class 'ibis.expr.operations.AnalyticVectorizedUDF'>`
I based the mark_only_on_backends annotations on the existing annotations for similar tests.

Reply:
kk

Reply:
Hmm. this seems weird rule for AnalyticVectorizedUDF